.yarnrc.yml

```diff
@@ -0,0 +1,3 @@
+nodeLinker: node-modules
+
+yarnPath: .yarn/releases/yarn-2.4.1.cjs
```

Comment:
I can't see the file yarn-2.4.1.cjs

What is in there? Can it be stored in substrate-js-dev so we can just update it in one place?

Reply:
I can see the file name in the files changed tab but, there is nothing as well under it for me. But if you pull down the branch the script will show up. Just checked it out. So it must be some kind of github characteristic. 

> Can it be stored in substrate-js-dev so we can just update it in one place?

I am not sure, but I would caution on the side of we probably shouldn't. Since yarn 2.x was built to be a per repo basis package manager, I am not sure how it would interact without having the script locally and what kind of side effects it could cause.